rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/AccessTypeRolesController.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.ccd.definition.store.rest.endpoint;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.ApiOperation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import uk.gov.hmcts.ccd.definition.store.repository.model.OrganisationProfileIds;
+import uk.gov.hmcts.ccd.definition.store.repository.model.AccessTypeRolesJurisdictionResult;
+import uk.gov.hmcts.ccd.definition.store.repository.model.AccessTypeRolesJurisdictionResults;
+import uk.gov.hmcts.ccd.definition.store.rest.service.AccessTypeRolesService;
+
+import javax.validation.Valid;
+
+@RestController
+@Api(value = "/api")
+@RequestMapping(value = "/api")
```

Comment:
minor issue - I dont really know why we added /api as root here, but if ccd are OK with that, then its fine

Reply:
Can this be removed? as it doesn't match the ticket description or any corresponding documentation you may have